src/pages/index.js

```diff
@@ -133,7 +133,7 @@ class Home extends Component {
                         fontSize: 30,
                       },
                     }}>
-                    A JavaScript library for building user interfaces
+                    biblioteka javascriptowa do tworzenia interfejsów użytkownika
```

Comment:
Jako, że to nowe zdanie:
![image](https://user-images.githubusercontent.com/7079786/53360026-c5bb9b80-3934-11e9-867b-1e1b917c0e24.png)
```suggestion
                    Biblioteka javascriptowa do tworzenia interfejsów użytkownika
```